EIPS/eip-7007.md

```diff
@@ -0,0 +1,160 @@
+---
+eip: 7007
+title: zkML AIGC-NFTs
+description: An ERC-721 extension interface for zkML based AIGC-NFTs.
+author: Cathie So (@socathie), Xiaohang Yu <xhyumiracle@hyperoracle.io>, Huaizhe Xu (@HuaizheXu), Kartin <kartin@hyperoracle.io>
+discussions-to: https://ethereum-magicians.org/t/eip-7007-zkml-aigc-nfts-an-erc-721-extension-interface-for-zkml-based-aigc-nfts/14216
+status: Draft
+type: Standards Track
+category: ERC
+created: 2023-05-10
+requires: 165, 721
+---
+
+## Abstract
+
+The zkML AIGC-NFTs standard is an extension of the [ERC-721](./eip-721.md) token standard for AI Generated Content. It proposes a set of interfaces for basic interactions and enumerable interactions for AIGC-NFTs. The standard includes a new mint event and a JSON schema for AIGC-NFT metadata. Additionally, it incorporates zkML capabilities to enable verification of AIGC-NFT ownership. In this standard, the `tokenId` is indexed by the `prompt`.
```

Comment:
Please expand abbreviations the first time they are used (eg. `The zebra kebab making license (zkML)...`)